Gemfile

```diff
@@ -13,5 +13,5 @@ group :test do
   gem "gems"
   gem "html-proofer"
   gem "rspec"
-  gem "rubocop", "~> 0.51"
+  gem "rubocop", "~> 0.52"
```

Comment:
isn't this change unnecessary? I mean wouldn't `~> 0.51` automatically resolve to the latest version of Rubocop?

Reply:
This disallows `0.51` explicitly because reasons